R/ltcad.R

```diff
@@ -0,0 +1,133 @@
+#' @method epidist_prepare epidist_ltcad
```

Comment:
Need issues to flesh out the docs for these methods

Reply:
Added issue for this https://github.com/epinowcast/epidist/issues/74